mysql/datadog_checks/mysql/statements.py

```diff
@@ -167,20 +147,20 @@ def _query_summary_per_statement(self, db):
         """
 
         sql_statement_summary = """\
-            SELECT `schema_name` as `schema`,
-                `digest` as `digest`,
-                `digest_text` as `query`,
-                `count_star` as `count`,
-                `sum_timer_wait` / 1000 as `time`,
-                `sum_lock_time` / 1000 as `lock_time`,
-                `sum_errors` as `errors`,
-                `sum_rows_affected` as `rows_affected`,
-                `sum_rows_sent` as `rows_sent`,
-                `sum_rows_examined` as `rows_examined`,
-                `sum_select_scan` as `select_scan`,
-                `sum_select_full_join` as `select_full_join`,
-                `sum_no_index_used` as `no_index_used`,
-                `sum_no_good_index_used` as `no_good_index_used`
+            SELECT `schema_name`,
+                   `digest`,
+                   `digest_text`,
+                   `count_star`,
+                   `sum_timer_wait`,
+                   `sum_lock_time`,
```

Comment:
is it intentional to remove the `/ 1000` for times?

Reply:
Yes, we're moving scaling operations like this to the backend.